README.md

```diff
@@ -9,26 +9,14 @@ _Installs dependencies from a local checkout, and keeps them in sync, without th
 
 # Summary
 
-Relative deps introduces an additional dependency section in `package.json`, called `relativeDependencies`.
-This section contains paths to the local sources of any dependency, that will be built and installed over the publicly available versions, when needed.
+Relative deps introduces an additional dependency section in a config file called `relative-deps.json`
+This config contains paths to the local sources of any dependency, that will be built and installed over the publicly available versions, when needed.
 
 Example `package.json`:
 
 ```json
 {
-  "name": "my-project",
-  "dependencies": {
-    "my-cool-library": "0.1.0"
-  },
-  "relativeDependencies": {
-    "my-cool-library": "../../packages/my-cool-library"
-  },
-  "scripts": {
-    "prepare": "relative-deps"
-  },
-  "devDependencies": {
-    "relative-deps": "^1.0.0"
-  }
+  "my-cool-library": "../../packages/my-cool-library"
```

Comment:
1. I don't understand this. It looks like a property at the top level of the `package.json` file. It looks like a mistake. How will `relative-deps` find this dep?

2. I think we should offer both the old and new ways of declaring relative dependencies in the readme. Some prefer to keep everything in package.json, while others want to separate. Unless we're deprecating the former, it should be documented and clearly available.

Reply:
Regarding the first comment- it's indeed a mistake. it's an example of the `relative-deps.json`.
About the second comment- IMHO, I think we should deprecate the old way of putting the config in package.json but still support people who have it there (which is what I did in this PR)